src/Symfony/Component/Security/Core/Tests/User/InMemoryUserProviderTest.php

```diff
@@ -25,6 +25,7 @@ public function testConstructor()
         $user = $provider->loadUserByUsername('fabien');
         $this->assertEquals('foo', $user->getPassword());
         $this->assertEquals(['ROLE_USER'], $user->getRoles());
+        $this->assertEquals(['name' => 'John', 'age' => 77, 'salary' => 1234.56], $user->getExtraFields());
```

Comment:
assertSame ?

Same for the others in this file

Reply:
I do not know if it is related to this PR to change the method name

and for this new added line I just duplicate the above line to keep things similar

if asked/decided I can handle it inside this PR of course :)